src/ui.rs

```diff
@@ -69,14 +95,18 @@ pub struct State<'a> {
     pub render_infobar: bool,
     /// render_help determines whether or not the help info should be rendered.
     pub render_help: bool,
-    /// Should the image shown be shown in actual pixel dimensions
-    pub actual_size: bool,
     /// Tracks fullscreen state of app.
     pub fullscreen: bool,
     /// current mode of the application, changes how input is interpreted
     pub mode: Mode,
     /// last_action records the last action performed. Used for repeating that action
     pub last_action: Action<'a>,
+    /// scale represents the scale of the image with 1.0 being the actual size of the image
+    pub scale: f32,
+    /// pan_x is the degree of pan in the x axis
```

Comment:
degree (like angles)?

Hard to think of a word to describe the percentage off in a direction from center of image.